exercises/bracket-push/bracket_push_test.rb

```diff
@@ -1,43 +1,81 @@
+#!/usr/bin/env ruby
 gem 'minitest', '>= 5.0.0'
 require 'minitest/autorun'
 require_relative 'bracket_push'
 
-class BracketPushTest < Minitest::Test
+# Test data version:
+# 4626617 Merge pull request #237 from moofkit/bracket_push_tests
+
+class BracketsTest < Minitest::Test
+  def test_paired_square_brackets
+    assert Brackets.paired?('[]')
+  end
+
   def test_empty_string
+    skip
     assert Brackets.paired?('')
   end
 
-  def test_pair_brackets
+  def test_unpaired_brackets
     skip
-    assert Brackets.paired?('{}')
+    refute Brackets.paired?('[[')
   end
 
-  def test_unpaired
+  def test_wrong_ordered_brackets
     skip
-    refute Brackets.paired?('[[')
+    refute Brackets.paired?('}{')
+  end
+
+  def test_paired_with_whitespace
+    skip
+    assert Brackets.paired?('{ }')
   end
 
-  def test_wrong_ordered
+  def test_simple_nested_brackets
     skip
-    refute Brackets.paired?(')(')
+    assert Brackets.paired?('{[]}')
   end
 
-  def test_paired_nested
+  def test_several_paired_brackets
+    skip
+    assert Brackets.paired?('{}[]')
+  end
+
+  def test_paired_and_nested_brackets
     skip
     assert Brackets.paired?('([{}({}[])])')
   end
 
-  def test_unpaired_nested
+  def test_unpaired_and_nested_brackets
     skip
     refute Brackets.paired?('([{])')
   end
 
-  def test_wrong_nested
+  def test_paired_and_wrong_nested_brackets
     skip
     refute Brackets.paired?('[({]})')
   end
 
-  # Don't forget to define a constant VERSION inside of Brackets.
+  def test_math_expression
+    skip
+    assert Brackets.paired?('(((185 + 223.85) * 15) - 543)/2')
+  end
+
+  def test_complex_latex_expression
+    skip
+    # rubocop:disable LineLength
+    assert Brackets.paired?('\left(\begin{array}{cc} \frac{1}{3} & x\\ \mathrm{e}^{x} &... x^2 \end{array}\right)')
+    # rubocop:enabled LineLength
```

Comment:
My thought was that we could still have a `LaTex` string but it would be shortened down so it would fit on a single line. Depends on how much we care about line length vs keeping everything in this example.